deploy/release.yaml

```diff
@@ -4,11 +4,15 @@ kind: ServiceAccount
 metadata:
   name: knative-openshift-ingress
   namespace: knative-serving
+  labels:
+    serving.knative.dev/release: v0.0.6
```

Comment:
We used `serving.knative.openshift.io` instead of `serving.knative.dev` for the annotation - https://github.com/openshift-knative/knative-openshift-ingress/blob/master/pkg/controller/resources/route.go#L19-L20
We will use `serving.knative.dev` here?

In the beginning, I wanted to output the release label by this command, but I'm wondering whether we should `serving.knative.dev` or `serving.knative.openshift.io` now.

```
oc get deploy -n knative-serving --label-columns=serving.knative.dev/release
```

Reply:
That's where I had second thoughts as well 🤔